.changeset/tiny-buses-compete.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': minor
```

Comment:
Let's make this a `patch` change, since we're not breaking anything and we're on 0.x